### PR TITLE
Ga usability topics update

### PR DESCRIPTION
**Pull request by ig-georgeA**
Updated the docs and images for following three topics:

- creating a prototype
- setting-up-a-usability-study
- usability-study-results-overview
- workspaces
- interaction options

### CODE REVIEW

**Comment by ig-georgeA** [comment]
@StefanIvanov  will the docs fail to compile without the following format for images?

`<div class="divider--half"></div>

<div class="divider--half"></div>
<div class="divider--half"></div>
<div class="divider--half"></div>
<div class="divider--half"></div>
<div class="divider--half"></div>`

I typically do the following for images in markdown to keep it readable when editing.
![][img-1]
<Br/>

[img-1]: ../images/commenting_2.png

**Comment by StefanIvanov** [comment]
@ig-georgeA the first two topics creating-a-prototype and interaction-options look fine and there are no problems with them, everything runs and images are loaded correctly. I see you have removed some dividers, I use them to create some visual hierarchy when there is an image, short text, and another image because sometimes it looks confusing which image the text is referring to at first glance. If you feel that the topic is readable, then you can remove dividers. I saw that for the other three topics it is still a work in progress and the images are missing so I am not going to make a review just yet.

**Comment by ig-georgeA** [comment]
@StefanIvanov ok. Now I understand what the dividers are for.

About your other comment, I have added images for all topics. But Github sometimes recognizes that way I'm including images, and sometimes it does not. Wondering whether you had some insights.

**Comment by StefanIvanov** [comment]
@ig-georgeA I use sourcetree and drag-drop the images once I open the project in VS Code in the project tree, where the images folders are

File: en/prototyping/usability-study-results-overview.md [hunk 1]
```diff
@@ -137,8 +108,24 @@ Watch some of the videos we have on usability studies.
...
+
+[topic-1]: setting-up-a-usability-study.md
+
+[img-1]: ..images/understanding_the_usability_test_report_1.png
+[img-2]: ..images/understanding_the_usability_test_report_2.png
+[img-3]: ..images/understanding_the_usability_test_report_3.png
+[img-4]: ..images/understanding_the_usability_test_report_4.png
+[img-5]: ..images/understanding_the_usability_test_report_5.png
+[img-6]: ..images/understanding_the_usability_test_report_6.png
+[img-7]: ..images/understanding_the_usability_test_report_7.png
```
Comment (StefanIvanov) [inline en/prototyping/usability-study-results-overview.md:130]: You also need to fix the images here

Comment (ig-georgeA) [reply en/prototyping/usability-study-results-overview.md:130]: I updated how I reference images now @StefanIvanov. All images should be showing now.

**Comment by StefanIvanov** [comment]
OK I have merged it, once we see it on staging you and @pmoleri can make a final review and localization will create a separate PR for JP so leave the label up there

**Pull request merged**